modules/admin-ui/src/main/webapp/scripts/shared/partials/hotkeyCheatSheet.html

```diff
@@ -0,0 +1,28 @@
+<div ng-class="{'modal-animation modal-overlay': helpVisible}"></div>
+<div ng-controller="HotkeyCheatSheetCtrl" ng-show="helpVisible" class="modal modal-animation">
+  <header>
+    <a class="fa fa-times close-modal" ng-click="toggleCheatSheet()"></a>
+    <h2 translate="HOTKEYS.CHEAT_SHEET.TITLE"><!-- Keyboard Shortcuts --></h2>
+  </header>
+  <div class="modal-content active"><div class="modal-body"><div class="full-col">
```

Comment:
Special reason to have both div on the same line?

Reply:
Just a stylistic choice. I don't like deep nesting, especially if the "layers" don't contribute much to the understanding of main purpose of the piece of code (see also our discussion about single line `if`-s :wink:). This line just contains all the necessary `div`-s and `class`-es to make the whole thing "modal-content-like". I can split them if you want, though; my opinion on this one is less rigid than the one on the `if` issue. x)